README.md

```diff
@@ -88,7 +88,7 @@ $ source activate cudf_dev
 ```
 
 - Build and install `librmm` using cmake & make. CMake depends on the `nvcc` executable being on
-  your path or defined in `$CUDACXX`.
+  your path or defined in `CUDACXX` environment variable.
```

Comment:
@kkraus14 I know see that this doc changes are found in other RAPIDS project. I now think I should drop this change, so the docs are consistent across all projects. 

Thoughts?

Reply:
We should just fix them across the other projects as well moving forward.